spec/models/user_spec.rb

```diff
@@ -1,5 +1,41 @@
-# require 'rails_helper'
-#
-# RSpec.describe User, :type => :model do
-#   pending "add some examples to (or delete) #{__FILE__}"
-# end
+require 'rails_helper'
+
+RSpec.describe User, :type => :model do
+  context 'Create new user.' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.